### PR TITLE
cherrypick for PLUGIN-1654: Netezza BugFix for 1.7 release

### PR DESCRIPTION
**Pull request by Shubhangi-cs**
CDAP : https://cdap.atlassian.net/browse/PLUGIN-1654
Pipeline failing for 52000 record Load - Netezza
Cherrypicked from this PR: https://github.com/data-integrations/database-plugins/pull/423

### CODE REVIEW

File: database-commons/src/main/java/io/cdap/plugin/db/batch/sink/ETLDBOutputFormat.java [hunk 1]
```diff
@@ -89,8 +89,8 @@ public void close(TaskAttemptContext context) throws IOException {
           try {
             if (!emptyData) {
```
Comment (sau42shri) [inline database-commons/src/main/java/io/cdap/plugin/db/batch/sink/ETLDBOutputFormat.java:90]: I didn't understand this part. There was a 1.10 cherry pick. Why are we picking it up again for 1.7. Is there compatibility issue with different CDF versions for 1.10

Comment (Shubhangi-cs) [reply database-commons/src/main/java/io/cdap/plugin/db/batch/sink/ETLDBOutputFormat.java:90]: Yes, we are releasing it with two different CDF versions.

**Comment by sau42shri** [comment]
Approved.

**Comment by sau42shri** [comment]
Approved.

**Review by sau42shri — APPROVED** [review]
Approved.

**Pull request merged**